src/lib/RoadieAgentReceiver.ts

```diff
@@ -76,4 +83,38 @@ export class RoadieAgentReceiver {
       );
     });
   }
+
+  private registerScaffolderAction(
+    configuration: ScaffolderActionAgentConfiguration,
+    app: Express,
+  ) {
+    // Specifying routes explicitly
+    app.get(`/scaffolder-action/${configuration.name}`, (req, res) => {
```

Comment:
Entrypoint to the Agent functionality. Triggers the action by creating a context and helper wrapper for it, within `CustomScaffolderAction`